themes/_global/js/autoClose.js

```diff
@@ -1,6 +1,26 @@
 $(document).ready(function() {
 	current_url = window.location.href
 	if (current_url.match("&closewin=true")) {
+		if (opener) {
```

Comment:
Yeah, I can rewrite this to `window.opener` if you want

And if you allow me, I think I could make the surrounding JS a bit nicer:
```diff
-	current_url = window.location.href
-	if (current_url.match("&closewin=true")) {
+	if (location.search.match("&closewin=true")) {
```